proxy_ring_buffer.go

```diff
@@ -0,0 +1,278 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package hazelcast
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	hzcerrors "github.com/hazelcast/hazelcast-go-client/internal/hzerrors"
```

Comment:
By convention, we add prefix `i` to the internal import aliases (just like the iserialization import below). So this import would become:
```
ihzerrors "github.com/hazelcast/hazelcast-go-client/internal/hzerrors"
```

Reply:
Sure, done.
see c197bec31d867b265efa486072ff4fed1a9db6cf

Reply:
I took the freedom to change it in some more places, which did not follow that convention.